src/twisted/newsfragments/9982.misc

```diff
@@ -0,0 +1 @@
+Exempt ``__repr__``s from compatability policy.
```

Comment:
There is a `doc` category for documentation changes, that may be more appropriate than `misc` since this is not a trivial change.

Also "compatability" -> "compatibility" here as well.

Reply:
```suggestion
Exempt ``__repr__``s from compatibility policy.
```